gov/agendas/deployments.go

```diff
@@ -252,8 +253,9 @@ func (db *AgendaDB) storeAgenda(agenda *AgendaTagged) error {
 
 // UpdateAgendas updates agenda data for all configured vote versions.
 func (db *AgendaDB) UpdateAgendas() error {
-	if db == nil || db.sdb == nil {
-		return errDefault
```

Comment:
I didn't see a good reason for these. The only time this would be the case is if an error is returned from the constructor, in which case the main loop returns anyway.